opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -484,6 +484,75 @@ namespace Opm {
             return istlSolver().iterations();
         }
 
+
+        void makeOverlapRowsInvalid(Mat& ebosJacIgnoreOverlap) const
+        {
+	  //local id's of cells
+	  auto lid = grid_.localIdSet();
+	  
+	  //value to set on diagonal
+	  double diag_val = 1.0e100;
+	  
```

Comment:
Create an object if MatrixBlockType, typedef is already there in class, and set that to diag( 1e100).